server/main.rs

```diff
@@ -1,3 +1,22 @@
+#![feature(core)]
+#![feature(io)]
+
+extern crate "hematite_server" as hem;
+
+use std::old_io::MemReader;
+
+use hem::packet::Protocol;
+
 fn main() {
-    println!("Hello, world!");
+	let value = false;
```

Comment:
That's Sublime being a douche... I don't why but it mixes tabs and spaces only on Rust files.